packages/components/src/organisms/OutOfStock/OutOfStockTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react'
+
+export type OutOfStockTitleProps = {
```

Comment:
react-docgen-typescript works better with an interface to extract the props automatically.

```suggestion
export interface OutOfStockTitleProps = {
```